content/java/concepts/array-list/terms/indexOf/indexOf.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: '.indexOf()'
+Description: 'Returns the index of the first occurrence of a given element, or -1 if not found.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+  - 'Elements'
+  - 'Search'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.indexOf()`** method returns the index of the first occurrence of the specified element in an `ArrayList`. If the element is not found, -1 is returned.
+
+## Syntax
+
+```pseudo
+myArrayList.indexOf(element);
+```
+
+The index of its first occurrence is returned if the `element` exists, even if the value is `null`. If the `element` can't be found, -1 will be returned.
+
+## Example
+
+The following example features two calls to `.indexOf()` on an `ArrayList` called `animals`. The first call will print the index of the first occurrence of the element with the string `"Tiger"`. The second will return -1 because an element with the string `"Elephant"` does not exist:
```

Comment:
```suggestion
The following example features two calls to `.indexOf()` on an `ArrayList` called `animals`. The first call will print the index of the first occurrence of the element  `"Tiger"`. The second will return -1 because the element `"Elephant"` does not exist:
```